server/database/queries/users/searchUsers.js

```diff
@@ -1,9 +1,9 @@
 const dbConnection = require('../../config/dbConnection');
 
-exports.searchUsers = () => {
+exports.searchUsers = (query) => {
   const sql = {
-    text: '',
-    values: [],
+    text: 'SELECT * FROM users WHERE username LIKE $1 ',
```

Comment:
btw, if I'm not mistaken, the users table stores the passwords correct? Selecting every column and then passing the data to the response will send all of the selected users' passwords to the client! That's a pretty big security flaw (even thogh the passwords are hashed, you most certainly do not want to expose them :) 
This is why it's generally good to store passwords in a separate table (even better, use 3rd party authentication providers). As a quick fix, you can simply exclude the password column from the query (this should be done for all queries from the user table..)

Reply:
WAIT UR RIGHT LMAO
BIG THANKS